cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Genesis.hs

```diff
@@ -280,16 +278,26 @@ genesisUtxOs sg =
 initialFundsPseudoTxIn :: ShelleyAddress -> ShelleyTxIn
 initialFundsPseudoTxIn addr =
     case addr of
-      Shelley.Addr _networkId (Shelley.KeyHashObj    (Shelley.KeyHash    h)) _sref -> pseudoTxIn h
-      Shelley.Addr _networkId (Shelley.ScriptHashObj (Shelley.ScriptHash h)) _sref -> pseudoTxIn h
+      Shelley.Addr _networkId (Shelley.KeyHashObj    (Shelley.KeyHash    h)) _sref -> pseudoTxInAddr h
+      Shelley.Addr _networkId (Shelley.ScriptHashObj (Shelley.ScriptHash h)) _sref -> pseudoTxInHash h
       Shelley.AddrBootstrap byronAddr ->
         panic $ "Unsupported Byron address in the genesis UTxO: " <> show byronAddr
   where
-    pseudoTxIn :: Crypto.Hash (HASH TPraosStandardCrypto) a -> ShelleyTxIn
-    pseudoTxIn h = Shelley.TxIn (pseudoTxId h) 0
+    pseudoTxInAddr :: Crypto.Hash (ADDRHASH TPraosStandardCrypto) a -> ShelleyTxIn
+    pseudoTxInAddr h = Shelley.TxIn (pseudoTxIdAddr h) 0
```

Comment:
I couldn't find a way to combine `pseudoTxInAddr` and `pseudoTxInHash` as a single polymorphic function.

Reply:
That functionality has moved to `ledger-specs` and I should be using that.

Reply:
Will do that in an upcoming PR.